modules/Microsoft.Web/sites/.test/wa/deploy.test.bicep

```diff
@@ -0,0 +1,101 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.web.sites-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'wswa'
```

Comment:
It could be tricky to understand what the test is about from this name/folderpath. I know it's still an open discussion, but what would you think about having the following:

Subfolder:functionappcommon
Serviceshort:wsfacom

Subfolder:functionappmin
Serviceshort:wsfamin

Subfolder:webappcommon
Serviceshort:wswacom

Subfolder:webappmin
Serviceshort:wswamin

Reply:
Fair enough. Updated.

Will soon open a follow up update to the Set-ModuleReadMe script though, as the 'Deployment Examples' in the ReadMe are now written as `<h3>Example 1: Functionappcommon</h3>`. I'd want to instead split the name at every upper character. So for example: `<h3>Example 1: Function App Common</h3>`

Reply:
Agree, that would be great